omnibus/files/private-chef-cookbooks/private-chef/recipes/config.rb

```diff
@@ -51,6 +51,7 @@
 
   if chef_server_rb_exists
     chef_server_rb_not_empty = !File.zero?(chef_server_path)
+    FileUtils.chmod('o-rwx', chef_server_path)
```

Comment:
Using File.chmod requires permissions to be set in absolute mode. I figured it was more pragmatic to remove 'other' permissions than to explicitly set all permission for owner, group, and other.

I agree on moving it down after symlinks and whatnot.